src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerGeneralEmail.java

```diff
@@ -96,6 +99,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .mandatory(GeneralEmail::getGeneralEmailOtherRecipientEmail, "generalEmailParties=\"other\"")
             .mandatory(GeneralEmail::getGeneralEmailOtherRecipientName, "generalEmailParties=\"other\"")
             .mandatory(GeneralEmail::getGeneralEmailDetails)
+            .label("attachmentWarning", WARNING_ATTACHMENT_SIZE)
             .optional(GeneralEmail::getGeneralEmailAttachments)
             .done();
     }
```

Comment:
We could also check whether there's an attachment and then show a warning on event submission:
<img width="587" alt="image" src="https://github.com/user-attachments/assets/67d7fb87-816e-4f1b-ac29-f1cacb4e32ce">
If there's a warning in the callback response, it shows them the warning message and they have to confirm event submission by submitting again.